articles/azure/azs-how-setup-recovery-portal.md

```diff
@@ -0,0 +1,486 @@
+---
+title: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+description: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+services: azure-stack
+author: David Woffendin
+toc_rootlink: Users
+toc_sub1: How To
+toc_sub2: Site Recovery
+toc_sub3:
+toc_sub4:
+toc_title: How to setup disaster recovery for Azure Stack VMs to Azure via the portal
+toc_fullpath: Operators/How To/azs-how-setup-recovery-portal.md
+toc_mdlink: azs-how-setup-recovery-portal.md
+---
+# Setting up disaster recovery for Azure Stack VMs to Azure via the portal
+
+## Overview
+
+The purpose of this article is to help with the setting up of the **Azure Stack** disaster recovery to **Public Azure** using the **Azure Site Recovery Service**.
```

Comment:
This article shows you how to set up Azure Stack disaster recovery to Public Azure, using the Azure Site Recovery Service.

Reply:
done